paper.tex

```diff
@@ -281,7 +293,7 @@ \subsection{Fleet Reactor Implementation}
 
 \subsection{Scenario Description}
 
-The scenario modeled approximates a transition from a the current once-through light water
+The chosen scenario approximates a transition from a the current once-through light water
 reactor (LWR) fleet to a full sodium fast reactor (SFR) fuel cycle within evaluation group 23 (EG-23)
 in the DOE's Advanced Nuclear Fuel Cycle Options report
 \cite{wigeland_nuclear_2014}. Scenario and transition details were patterned
```

Comment:
Not sure if we should make such an explicit reference to this.  On one hand, it reveals things they are secretive about.  On the other hand, we should give them due credit.